modules/navier_stokes/test/tests/finite_volume/controls/switch-pressure-bc/switch_vel_pres_bc.i

```diff
@@ -0,0 +1,382 @@
+    advected_interp_method = ${advected_interp_method}
+    rho = ${rho}
+    momentum_component = 'y'
+  []
+  [v_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = v
+    mu = ${mu}
+    momentum_component = 'y'
+  []
+  [v_pressure]
+    type = INSFVMomentumPressure
+    variable = v
+    momentum_component = 'y'
+    pressure = pressure
+  []
+
+  [temp_time]
+    type = WCNSFVEnergyTimeDerivative
+    variable = T
+    rho = rho
+    drho_dt = drho_dt
+  []
+  [temp_conduction]
+    type = FVDiffusion
+    coeff = 'k'
+    variable = T
+  []
+  [temp_advection]
+    type = INSFVEnergyAdvection
+    variable = T
+    velocity_interp_method = ${velocity_interp_method}
+    advected_interp_method = ${advected_interp_method}
+  []
+  [heat_source]
+    type = FVCoupledForce
+    variable = T
+    v = power_density
+  []
+[]
+
+[FVBCs]
+  # Inlet
+  [inlet_u]
+    type = WCNSFVSwitchableInletVelocityBC
+    variable = u
+    boundary = 'left'
+    mdot_pp = 'inlet_mdot'
+    area_pp = 'surface_inlet'
+    rho = 'rho'
+    switch = true
+    face_limiter = 1.0
+  []
+  [outlet_u]
+    type = WCNSFVSwitchableInletVelocityBC
+    variable = u
+    boundary = 'right'
+    mdot_pp = 'inlet_mdot'
+    area_pp = 'surface_inlet'
+    rho = 'rho'
+    switch = false
+    scaling_factor = -1.0
+    face_limiter = 1.0
+  []
+
+  [inlet_v]
+    type = WCNSFVInletVelocityBC
+    variable = v
+    boundary = 'left'
+    mdot_pp = 0
+    area_pp = 'surface_inlet'
+    rho = 'rho'
+  []
+
+  [inlet_T]
+    type = WCNSFVInletTemperatureBC
+    variable = T
+    boundary = 'left'
+    temperature_pp = 'inlet_T'
+  []
+  [outlet_T]
+    type = NSFVOutflowTemperatureBC
+    variable = T
+    boundary = 'right'
+    u = u
+    v = v
+    rho = 'rho'
+    cp = 'cp'
+    backflow_T = ${inlet_temp}
+  []
+
+  [outlet_p]
+    type = INSFVSwitchableOutletPressureBC
+    variable = pressure
+    boundary = 'right'
+    function = ${outlet_pressure}
+    switch = true
+    face_limiter = 1.0
+  []
+  [inlet_p]
+    type = INSFVSwitchableOutletPressureBC
+    variable = pressure
+    boundary = 'left'
+    function = ${outlet_pressure}
+    switch = false
+    face_limiter = 1.0
+  []
+
+  # Walls
+  [no_slip_x]
+    type = INSFVNoSlipWallBC
+    variable = u
+    boundary = 'top bottom'
+    function = 0
+  []
+  [no_slip_y]
+    type = INSFVNoSlipWallBC
+    variable = v
+    boundary = 'top bottom'
+    function = 0
+  []
+[]
+
+[Functions]
+  [func_coef]
+    type = ParsedFunction
+    expression = 'if(t<${switch_time} | t>2.0*${switch_time}, 1, 0)'
+  []
+  [func_coef_comp]
+    type = ParsedFunction
+    expression = 'if(t<${switch_time} | t>2.0*${switch_time}, 0, 1)'
+  []
+  [mass_flux_and_pressure_test_scaling]
+    type = ParsedFunction
+    expression = 'if(t<${switch_time} | t>2.0*${switch_time}, 0.1, 0.2)'
+  []
+[]
+
+[Controls]
+  [func_control_u_inlet]
+    type = BoolFunctionControl
+    parameter = 'FVBCs/inlet_u/switch'
```

Comment:
if you called it something more definite than just switch you could rely on this behavior for bool function control:
```
  params.addRequiredParam<std::string>(
      "parameter",
      "The input parameter(s) to control. Specify a single parameter name and all "
      "parameters in all objects matching the name will be updated");
```

Reply:
actually you would need to use a 'flip' behavior, because they have the same name but you are using different values

Reply:
Thanks. Changes `switch` --> `switch_bc`